containers/codespaces-linux/test-project/test.sh

```diff
@@ -75,5 +75,10 @@ check "zsh" zsh --version
 # Check expected commands
 check "git-ed" [ "$(cat /home/codespace/.local/bin/git-ed.sh)" = "$(cat ./git-ed-expected.txt)" ]
 
+# Check that we can run a puppeteer node app.
+yarn add puppeteer
```

Comment:
Should we just run `yarn` or `yarn install` here since we already have the package in package.json and a yarn.lock file?